src/main/java/co/omise/models/schedules/Schedule.java

```diff
@@ -30,6 +32,9 @@
     @JsonProperty("next_occurrences")
     private ScopedList<Occurrence> nextOccurrences;
 
+    @JsonProperty("next_occurrence_dates")
+    private List<String> nextOccurrenceDates;
```

Comment:
Can we keep `nextOccurrenceDates` as Date instead String krub?

Reply:
Can I use Joda LocalDate? 
Because the format of this field has no time (yyyy-MM-dd)

Reply:
@thanapolr sounds good to me krub.

Reply:
I think String is ok for this case. Since there is no native LocalDate type in Java 1.7 yet

Reply:
Since we're already depend on Joda. Let's go with joda.LocalDate :)